src/app/components/search/SearchResultsCards/SharedItemCardFooter.js

```diff
@@ -1,116 +1,50 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { FormattedDate, FormattedMessage, injectIntl, intlShape } from 'react-intl';
+import { FormattedMessage } from 'react-intl';
 import BulletSeparator from '../../layout/BulletSeparator';
 import ButtonMain from '../../cds/buttons-checkboxes-chips/ButtonMain';
-import Tooltip from '../../cds/alerts-and-prompts/Tooltip';
-import MediaTypeDisplayIcon from '../../media/MediaTypeDisplayIcon';
-import MediaIcon from '../../../icons/perm_media.svg';
-import CalendarMonthIcon from '../../../icons/calendar_month.svg';
-import RequestsIcon from '../../../icons/question_answer.svg';
 import OpenInNewIcon from '../../../icons/open_in_new.svg';
 import ItemChannels from '../../cds/media-cards/ItemChannels';
-import { getCompactNumber, getSeparatedNumber } from '../../../helpers';
+import SuggestionsCount from '../../cds/media-cards/SuggestionsCount';
```

Comment:
Flagging for @caiosba that I refactored `SharedItemCardFooter`. Specifically the individual pieces are componentized, and for the counts we want to check `variable !== null` to see if we render, since `0` is a valid case here (we want to show "0 Requests" if we pass in `requestsCount={0}`

Reply:
Sounds good, thanks @dariusk! So let's please add smoke-testing feeds to the testing notes just to be sure that the refactoring didn't include any regressions.

Reply:
Ok! Added to the test plan